src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1332,7 +1332,7 @@ void goto_instrument_parse_optionst::instrument_goto_program()
   }
 
   // add generic checks, if needed
-  goto_check(options, goto_model);
```

Comment:
Use `ui_message_handler` here (should be the same as `log.get_message_handler()`, but is more consistent with the other code over here.

Reply:
corrected